package.json

```diff
@@ -18,8 +18,8 @@
     "prepublish": "npm run build",
     "prepublish-watch": "npm run watch",
     "start": "webpack-dev-server",
-    "tap": "./node_modules/.bin/tap ./test/unit/*.js",
-    "test": "npm run lint && npm run docs && npm run tap",
+    "tap": "./node_modules/.bin/tap ./test/unit/*.js test/integration",
```

Comment:
here we have `./test` ... followed by `test` is that intentional? The two should be consistent.

Reply:
hrm -- so they are slightly different - also this doesn't need to be this complex...

`tap test/unit test/integration` works just fine in an npm command the `./node_modules/.bin` is in path for those commands... I'll update this.